frontend/src/redux/reducers/menu.js

```diff
@@ -0,0 +1,10 @@
+import { SWITCH_MENU } from '../actionCreator';
+
```

Comment:
please create an initial state where the default value of 'currentlyRenderedComponent' (or something similar) is the 'Buildings', and you change that, based on the action.payload. 

```suggestion
const initialState = {
  currentlyRenderedComponent: 'Buildings',
};
```